src/components/HikeComposer/ExportModal.js

```diff
@@ -35,30 +39,30 @@ const ExportModal = ({ visible, jsonData, onClose, branchName, category }) => {
     }
   };
 
-  const visitFork = () => {
-    window.open(
-      `https://github.com/HCL-TECH-SOFTWARE/volt-mx-tutorials/compare/${BASE_BRANCH}...mikeangelsilva:${branchName}?expand=1`,
-      "_blank"
-    );
-  };
+  // const visitFork = () => {
+  //   window.open(
+  //     `https://github.com/HCL-TECH-SOFTWARE/volt-mx-tutorials/compare/${BASE_BRANCH}...mikeangelsilva:${branchName}?expand=1`,
+  //     '_blank',
+  //   );
+  // };
 
   return (
     <div>
       <Modal
         className={styles.exportModal}
         title={
           pushed
-            ? "Pushed Successfully"
+            ? 'Pushed Successfully'
```

Comment:
Minor comment. Certain strings in this file such as here have not been localised.

Reply:
Components under HikeComposer are not localised currently as it is a separate tool for editing tours, which is not a page seen by users on the catalog